cmd/crc/cmd/start.go

```diff
@@ -53,11 +54,14 @@ func runStart(arguments []string) {
 	}
 
 	commandResult, err := machine.Start(startConfig)
-	logging.InfoF(commandResult.Status)
 	if err != nil {
-		errors.ExitWithMessage(1, err.Error())
+		errors.Exit(1)
+	}
+	if commandResult.Status == "Running" {
```

Comment:
ideally this has to be checked against the State values.
but this will work for now...

Reply:
Yes, I have more changes in that area, but still not clear to me whether it's ok to remove `commandResult`, or if for now it's better to only improve the type of its fields (I'd be in favour of the former). I preferred to move forward with this first, instead of piling up too many changes :)